examples/drdecr/README.md

```diff
@@ -0,0 +1,316 @@
+# Training and Running Dr. DECR Models
+
+This file provides instructions on how to train and test the Dr. DECR models (Dense Retrieval with Distillation-Enhanced Cross-Lingual Representation) Student/Teacher pipeline, as desribed in [Learning Cross-Lingual IR from an English Retriever](https://arxiv.org/abs/2112.08185). 
+
+Similar models based on ColBERT V1, were used to obtain the results in Dr. DECR XOR-TyDi leaderboard whitebox (not using external APIs) submission.
+
+## Installation
+
+The following steps require PrimeQA to be [installed](../../README.md#Installation). 
+
+## Data Preparation
+
+There are three datasets used to train Dr. DECR:
```

Comment:
Are there citations for these datasets?

Reply:
Added a pointer to the section of the paper with the descriptions.

Reply:
Added citation about dataset.